### ISSUE TITLE
Upgrade to latest Ember LTS (3.20)

### ISSUE THREAD

**Issue by jlblcc** [adiwg/mdEditor#415]
https://cli.emberjs.com/release/basic-use/upgrading/
https://emberjs.com/releases/lts/

**Comment by dvonanderson** [comment]
- [ ] update forward- compatible addons 
- [ ] use ``` yarn upgrade-interactive --latest ``` to selectively update packages, except dependencies that are part of Ember
- [ ] use ``` ember-cli-update --to 3.16``` to push app to Ember 3.16
- [ ] add prettier to work with Eslint ``` yarn add -D eslint-plugin-prettier esling-config-prettier prettier ```
- [ ] configure .eslintrc.js 
```
extends: [
    'eslint:recommended',
    'plugin:ember/recommended',
    'plugin:prettier/recommended',
  ],
```
- [ ] use codemods in attempt to rewrite modules, components, and models
``` ember-cli-update --run-codemods ```

**Comment by dvonanderson** [comment]
closed via #426
